app/Http/Controllers/Rpc/WalletController.php

```diff
@@ -137,6 +140,38 @@ public function depositInfo()
         return self::json($resp);
     }
 
+    public function history()
+    {
+        $userId = Auth::user()->id;
+        $resp = [];
+        //TODO pagination
+        foreach (UserLedger::where('users_id', $userId)->cursor() as $ul) {
+            $amount = AdsConverter::clicksToAds($ul->amount);
+            $date = $ul->created_at->format(Carbon::RFC7231_FORMAT);
+            $txid = $ul->txid;
+            if (null !== $txid) {
+                // TODO export blockexplorer address / update testHistory in WalletControllerTest
+                $link = "https://operator1.e11.click/blockexplorer/transactions/${txid}";
```

Comment:
Please move Blockexplorer url to a dedicated variable in `.env`

Reply:
OK, I will move it.